script/Arcueid/Day 10/10_00_ARC10_3_2.txt

```diff
@@ -758,7 +758,7 @@ Acting on something other than reason,
 [sha:03f587b8f4427926d5879987ddf450356cee6db1]{
 -- Page 217, Offset 24821.
 -- 突然、見知らぬ第三者に声をかけられた。
-Suddenly, I am greeted by an unfamiliar third party.
+Suddenly, I am greeted by an unfamiliar third-party.
```

Comment:
Hypens only if used as an adjective (third-party plugin), but in this case she is simply a third person (party)